src/docs/components/modal-waitlist.hbs

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Modal Waitlist" }}
+</head>
+
+<body>
+	<div class="container mt-4">
+		<h1>Modal Waitlist</h1>
+    <button class="btn btn-primary" data-toggle="modal" data-target="#productWaitlist">Open Modal</button>
+    <div class="modal fade" id="productWaitlist" tabindex="-1" role="dialog" aria-labelledby="productWaitlist" aria-hidden="true">
+      <div class="modal-dialog modal-lg modal-dialog-centered waitlist-modal" role="document">
+        <div class="modal-content text-center bg-secondary-light text-secondary">
+          <div class="modal-header d-flex align-items-center flex-column border-bottom-0">
+            <button type="button" class="close sni sni__close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true"></span>
+            </button>
+            <p class="h1 mx-auto mb-1">Join the waitlist</p>
+            <p class="mb-0">Be the first to get the new Coco &amp; Pooch drop alert. Be quick, it will sell out fast!</p>
+          </div>
+          <div class="modal-body pt-0">
+            <form data-pdp="false" data-product-id="product-id">
+              <div class="row">
+                <div class="input-group form-group col">
+                  <input class="form-control" type="email" placeholder="Enter your email">
+                </div>
+              </div>
+              <span class="d-block mb-1 mb-lg-2">or</span>
+              <div class="row">
+                <div class="input-group form-group col-4 col-lg-2">
+                  <span class="masking-select position-absolute">Country</span>
+                  {{> components/countries-options }}
+                </div>
+                <div class="input-group form-group col-8 col-lg-10 pl-0">
+                  <input class="form-control" type="text" placeholder="Enter your phone number">
+                </div>
+              </div>
+              <div class="row">
+                <div class="input-group form-group col d-flex justify-content-center">
+                  <div class="custom-checkbox text-primary justify-content-center align-items-center mr-1 sni border rounded bg-white">
+                    <input type="checkbox" name="agree" class="opacity-0 position-absolute" required/>
+                  </div>
+                  <label class="form-check-label font-size-sm text-underline">I agree to Privacy Policy &amp; ToS</label>
```

Comment:
please always add attr FOR to labels, in this way clicking on it select/focus the field referred